.github/actions/setup/action.yml

```diff
@@ -4,6 +4,10 @@ description: "Sets up the environment with yarn, Node.js, and turbo"
 runs:
   using: "composite"
   steps:
+    - name: Enable Corepack
+      shell: bash
+      run: corepack enable
+
     - name: Get yarn cache directory path
```

Comment:
Since we're caching yarn here:

```
    - name: Setup Node.js
      uses: actions/setup-node@v4
      with:
        node-version-file: ".nvmrc"
        cache: "yarn"
```

Do we still need the manual caching with "Get yarn cache dir path"?